ng/js/directives/header.js

```diff
@@ -0,0 +1,36 @@
+require('angular').module('ays')
+
+  /* Author: Tyler Russell
+   * Directive to be placed in various HTML headers for user management,
+   * login/logout navigation, etc.
+   */
+  .directive('sessionheader', function (AuthService, UserService) {
+    return {
+      restrict: 'E',
+      controller: function ($scope, $sails) {
```

Comment:
Yeah I realized that. I made the changes in the most recent commit, along with all of the other changes you requested (bottom of this page) hash: 363c313